content/plugins/rupadana-swiper.md

```diff
@@ -0,0 +1,15 @@
+---
+name: Swiper
+slug: swiper
+author_slug: rupadana
+categories: [infolist-entry]
+description: The Most Modern Mobile Touch Slider now on Filamentphp
+discord_url: https://discord.com/channels/883083792112300104/1080807837833384017
+docs_url: https://raw.githubusercontent.com/rupadana/filament-swiper/main/README.md
+github_repository: rupadana/filament-swiper
+has_dark_theme: false
+has_translations: false
+image: rupadana-filament-swiper.png
```

Comment:
Remove 'image' key, this will be automatically generated.

But you do need to rename your image file to 'rupadana-swiper.png'